bower.json

```diff
@@ -32,7 +32,7 @@
     "paper-toggle-button": "PolymerElements/paper-toggle-button#1 - 2",
     "paper-styles": "PolymerElements/paper-styles#1 - 2",
     "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
-    "web-component-tester": "^4.0.0",
+    "web-component-tester": "^6.0.0",
```

Comment:
(((
needed to appease bower, otherwise it complains with:
```
Unable to find a suitable version for test-fixture, please choose one by typing one of the numbers below:
    1) test-fixture#^1.0.0 which resolved to 1.1.2 and is required by web-component-tester#4.3.6
    2) test-fixture#^3.0.0-rc.1 which resolved to 3.0.0-rc.1 and is required by app-localize-behavior)
```
)))

Reply:
🙌